centreon/src/EventSubscriber/CentreonEventSubscriber.php

```diff
@@ -357,14 +358,20 @@ public function onKernelException(ExceptionEvent $event): void
          * If we don't do that a HTML error will appeared.
          */
         if ($errorIsBeforeController) {
-            if ($event->getThrowable()->getCode() !== 403) {
-                $errorCode = $event->getThrowable()->getCode() > 0
-                    ? $event->getThrowable()->getCode()
-                    : Response::HTTP_INTERNAL_SERVER_ERROR;
+            if ($event->getThrowable()->getCode() >= Response::HTTP_INTERNAL_SERVER_ERROR) {
+                $errorCode = $event->getThrowable()->getCode();
                 $statusCode = Response::HTTP_INTERNAL_SERVER_ERROR;
-            } else {
+            } elseif ($event->getThrowable()->getCode() === Response::HTTP_FORBIDDEN) {
                 $errorCode = $event->getThrowable()->getCode();
                 $statusCode = Response::HTTP_FORBIDDEN;
+            } elseif ($event->getThrowable() instanceof NotFoundHttpException) {
+                $errorCode = Response::HTTP_NOT_FOUND;
+                $statusCode = Response::HTTP_NOT_FOUND;
+            } else {
+                $errorCode = $event->getThrowable()->getCode();
+                $statusCode = $event->getThrowable()->getCode() > 0
+                    ? $event->getThrowable()->getCode()
+                    : Response::HTTP_INTERNAL_SERVER_ERROR;
```

Comment:
If the code is not negative, this should be enough.
```suggestion
                $statusCode = $event->getThrowable()->getCode()
                    ?: Response::HTTP_INTERNAL_SERVER_ERROR;
```